README.md

```
@@ -1,36 +1,231 @@
+
+<br />
+
+## 📅 프로젝트 기간
+
+2026.02.02 ~ 2026.02.27
+
+<br />
+
+## 👥 팀원 소개
+
+|             <img src="public/assets/images/teamprofile/oneway.jpg" width="100" height="100" style="object-fit:cover;" />             |             <img src="public/assets/images/teamprofile/siwon.jpg" width="100" height="100" style="object-fit:cover;" />              |            <img src="public/assets/images/teamprofile/youvin.png" width="100" height="100" style="object-fit:cover;" />            |           <img src="public/assets/images/teamprofile/seohyun.jpg" width="100" height="100" style="object-fit:cover;" />           |
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)